common/src/main/resources/startup.properties

```diff
@@ -332,3 +333,6 @@ it.workflow.execution.listeners=org.apache.falcon.catalog.CatalogPartitionHandle
 #*.falcon.graphite.port=2003
 #*.falcon.graphite.frequency=1
 #*.falcon.graphite.prefix=falcon
+
+# Property to remove postProcessing
+*.falcon.postprocessing.enable=true
```

Comment:
Hi ,

This should be a startup.properties for couple of reasons like : 

We generate workflow.xml when the process is submitted or updated so if we keep on playing with runtime.properties user might be under the impression that next instance will not have post processing,if its a runtime properties how will we ensure the behavior across the two instance run.
 
Also if we change the runtime.properties we will change it for other users too who might not be expecting it to happen.